src/MyEventEmitter.js

```diff
@@ -1,12 +1,108 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  validName(eventName) {
+    if (typeof eventName !== 'string') {
+      return new Error('no valid event name');
+    }
+  }
+
+  validListener(listener) {
+    if (typeof listener !== 'function') {
+      return new Error('no valid event listener');
+    }
+  }
```

Comment:
Those methods aren't belong to MyEventEmitter, I would move it in separated class like Validator or smth like that